docs/api/Apify.md

```diff
@@ -197,7 +197,7 @@ actor</a> and several other API endpoints to obtain the output.
 <tr>
 <td colspan="3"><p>Specifies optional webhooks associated with the actor run, which can be used
  to receive a notification e.g. when the actor finished or failed, see
- <a href="https://apify.com/docs/webhooks#adhoc">ad hook webhooks documentation</a> for detailed description.</p>
+ <a href="https://docs.apify.com/webhooks#adhoc">ad hook webhooks documentation</a> for detailed description.</p>
```

Comment:
Could you also update the `#hash` part of the link? For example here:

`https://docs.apify.com/webhooks#adho`
=>
`https://docs.apify.com/webhooks/ad-hoc-webhooks`